vttools/to_wrap/image_processing/arithmetic.py

```diff
@@ -0,0 +1,271 @@
+             - When both 'x1' and 'x2' are of an integer type, 'divide' will
+               return integers and throw away the fractional part. Moreover,
+               division by zero always yields zero in integer arithmetic.
+        multiply:
+            The product of 'x1' and 'x2', element-wise. Returns a scalar if
+            both  'x1' and 'x2' are scalars.
+            Note: Equivalent to 'x1' * 'x2' in terms of array broadcasting.
+
+    x1, x2 : array
+        Can be floats or arrays
+
+    div_by_zero : {'ignore', 'warn', 'raise'}, optional
+        Treatment for division by zero.
+
+    out : array, optional
+        Array into which the output is placed. Its type is preserved and it
+        must be of the right shape to hold the output. See numpy doc.ufuncs.
+
+    Returns
+    -------
+    output : array  # use underscores for variable names, hyphens in prose
+        Returns the resulting array or constant to the designated variable
+
+    Example
+    -------
+    >>> x1 = np.array([[0, 1, 0], [1, 1, 1], [0, 1, 0]])
+    >>> x2 = np.array([[2, 0, 2], [0, 2, 0], [2, 0, 2]])
+    >>> arithmetic('add', x1, x2)
+    array([[2, 1, 2],
+           [1, 3, 1],
+           [2, 1, 2]])
+    >>> arithmetic('subtract', x1, x2)
+    array([[-2,  1, -2],
+           [ 1, -1,  1],
+           [-2,  1, -2]])
+    >>> arithmetic('multiply', x1, x2)
+    array([[0, 0, 0],
+           [0, 2, 0],
+           [0, 0, 0]])
+    >>> arithmetic('divide', x1, x2, div_by_zero='raise')
+    Traceback (most recent call last):
+      File "<stdin>", line 1, in <module>
+      File "/home/edill/dev/python/VTTools/vttools/to_wrap/arithmetic.py",
+      line 109, in arithmetic
+        return op(x1, x2, out)
+    FloatingPointError: divide by zero encountered in divide
+    >>> arithmetic('divide', x1, x2, div_by_zero='warn')
+    /home/edill/dev/python/VTTools/vttools/to_wrap/arithmetic.py:109:
+    RuntimeWarning: divide by zero encountered in divide
+      return op(x1, x2, out)
+    array([[0, 0, 0],
+           [0, 0, 0],
+           [0, 0, 0]])
+    >>> arithmetic('divide', x1, x2, div_by_zero='ignore')
+    array([[0, 0, 0],
+           [0, 0, 0],
+           [0, 0, 0]])
+    """
+    # ensure that inputs are numpy arrays
+    x1 = np.asarray(x1)
+    x2 = np.asarray(x2)
+    # use numpy built-in functionality to handle divide by zero problems
+    np.seterr(divide=div_by_zero)
+    # can use this one-liner instead of the mapping dictionary
+    op = getattr(np, operation)
+    return op(x1, x2, out)
+
+
+def arithmetic_expression(expression, A, B,
+                          C=None, D=None, E=None, F=None, G=None, H=None):
+    """Custom expression evaluator for up to 8 inputs A-H
+
+    Note that it would probably be a good idea (at some point!) to make use of
+    the 'Interpreter' object in lmfit.asteval as it appears to be a rather
+    parsing tool. @danielballan can speak to this better than I can
```

Comment:
Can you change this to say "@danielballan will be a good point of contact for this"?  As it stands, this was meant as a note from me to you